README.md

```diff
@@ -18,6 +18,26 @@ npm start
 
 Which will start the web application
 
+## Static build
+
+If you want a static build of the application you can execute
```

Comment:
"can" is filler word here. Remove it.

Reply:
Removed it in [commit 402db13](https://github.com/SolidLabResearch/generic-data-viewer/pull/62/commits/402db139419551cb472940c3ea22d759cad0df79)